src/codeFly/dispatcher/LoginHandler.java

```diff
@@ -0,0 +1,82 @@
+package codeFly.dispatcher;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import codeFly.CodeFly;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.util.HashMap;
+import java.util.ArrayList;
+
+/**
+ * A place holder for module that have not been implemented.

```

Comment:
Update the comment for this handler. The current one is for EchoRequestHandler only.